Makefile

```diff
@@ -187,6 +187,9 @@ $(replay_wasm): $(DEP_PREDICATE) $(go_source) .make/solgen
 	GOOS=js GOARCH=wasm go build -o $(output_root)/tmp/replay.wasm ./cmd/replay/...
 	if ! diff -qN $(output_root)/tmp/replay.wasm $@ > /dev/null; then cp $(output_root)/tmp/replay.wasm $@; fi
 
+arbitrator/prover/bulk_memory_internal.wasm: arbitrator/prover/bulk_memory_internal.wat
+	wat2wasm -o arbitrator/prover/bulk_memory_internal.wasm arbitrator/prover/bulk_memory_internal.wat
```

Comment:
We should consider making the generic `wasm => wat` rule cover this directory

Reply:
I didn't see an obvious clean way to do this, but I don't really understand the specific conventions in use in this Makefile so I'm happy to have you show me how you would do it!